hack/release.sh

```diff
@@ -114,6 +114,92 @@ s3_url() {
 	esac
 }
 
+map_goos_to_uname() {
+	# we need to map our GOOS to uname values
+	# see https://en.wikipedia.org/wiki/Uname
+	# ie, GOOS=linux -> "uname -s"=Linux
+	GOOS=$1
+	case "$GOOS" in
```

Comment:
Any particular reason to do this in two lines rather than just one? ie:

```bash
map_goos_to_uname() {
	# we need to map our GOOS to uname values
	# see https://en.wikipedia.org/wiki/Uname
	# ie, GOOS=linux -> "uname -s"=Linux
	case "$1" in
...
```